### ISSUE TITLE
Travis CI npm package warning: "No bin file found at dist/cli.js"

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#790]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug, Travis CI is reporting: `npm WARN package.json stylelint@4.4.0 No bin file found at dist/cli.js`

I thought it may have been related to #788 but as this has not been merged it should not be affecting PR's such as #780.

NodeJS 0.12.x and 4.x.x are affected for PR #780:
• Node 0.12: https://travis-ci.org/stylelint/stylelint/jobs/110679337#L141
• Node 4.x.x https://travis-ci.org/stylelint/stylelint/jobs/110679336#L143

NodeJS 5.x.x is _not_ affected for PR #780:
• Node 5.x.x https://travis-ci.org/stylelint/stylelint/jobs/110679335#L142 <- Not Applicable

**Comment by ntwb** [comment]
Diving back through previous Travis CI builds:

• Stylelint 4.3.0 & NodeJS 4.x.x affected: https://travis-ci.org/stylelint/stylelint/jobs/109063585#L96

**Comment by ntwb** [comment]
Looking at the `/dist` folder after running `npm run build`:
```sh
netweb@MacBook ~/dev/stylelint/stylelint/dist [master]
$ ls
total 48K
drwxr-xr-x  15 netweb staff  510 Feb 15 23:46 .
drwxr-xr-x  22 netweb staff  748 Feb 21 15:33 ..
drwxr-xr-x  11 netweb staff  374 Feb 15 23:46 __tests__
drwxr-xr-x   5 netweb staff  170 Jan 19 01:00 formatters
drwxr-xr-x 112 netweb staff 3.8K Feb 15 23:46 rules
drwxr-xr-x   5 netweb staff  170 Jan 19 01:00 testUtils
drwxr-xr-x  28 netweb staff  952 Feb 21 15:33 utils
-rw-r--r--   1 netweb staff 4.1K Feb 21 15:33 buildConfig.js
-rw-r--r--   1 netweb staff 4.1K Feb 21 15:33 cli.js
-rw-r--r--   1 netweb staff  184 Feb 21 15:33 createPlugin.js
-rw-r--r--   1 netweb staff 1.7K Feb 21 15:33 disableRanges.js
-rw-r--r--   1 netweb staff  906 Feb 21 15:33 index.js
-rw-r--r--   1 netweb staff 1.4K Feb 21 15:33 normalizeRuleSettings.js
-rw-r--r--   1 netweb staff 4.2K Feb 21 15:33 postcssPlugin.js
-rw-r--r--   1 netweb staff 4.5K Feb 21 15:33 standalone.js

netweb@MacBook ~/dev/stylelint/stylelint/dist [master]
```

Should `cli.js` have an `execute` attribute here?

**Comment by jeddy3** [comment]
I think it's fine. It's just a warning because the `/dist/cli.js` is built after the `npm install`. It's always been there, but I'm afraid I can't seem to find the historical issue where we first discussed it.

**Comment by davidtheclark** [comment]
That caused a problem for me at some point, but hasn't since. It was one of the first reasons I started regretting that the codebase is ES2015 instead of standard, uncompiled JS. :(

**Comment by jeddy3** [comment]
Shall we close this issue as appveyor and travis are green, and it's not currently impacting on our local development?

**Comment by ntwb** [comment]
Cool, thanks for the explanation :+1:
